### ISSUE TITLE
DRF: Model with blank=True leads to error in serializer.EnumChoiceField

### ISSUE THREAD

**Issue by SaturnFromTitan** [HackSoftware/django-enum-choices#45]
When setting `blank=True` in a model, the serializer implicitly inherits a serializer field with `allow_blank=True`. This, however, is being rejected by `django_enum_choices.serializers.EnumChoiceField` as an unknown argument in the `__init__` call (since it only inherits from the regular `serializers.Field`).

The work around I applied in my code is to add `allow_blank` to dump_kwargs [here](https://github.com/HackSoftware/django-enum-choices/blob/master/django_enum_choices/serializers.py#L101)

Do you agree this is the preferred solution? If so, I'm willing to submit a PR for it.

From an API perspective, I think it would be nicer to not need to specify blank=True at all (since you can derive that from the enum itself). However, it is needed as the admin menu of this model doesn't allow setting empty strings as the field on the admin menu form is marked as `required=True` otherwise. This might be a different discussion though.

**Comment by slavov-v** [comment]
Thanks for the catch! :v: 

I think `allow_blank` only makes sense on string based serializer fields.
It also works on choice fields, but if we were to adapt it in `EnumChoiceField` we would need some way to enforce that the `enum_class` passed to the model field has a defined blank option that the field can use when it receives an empty string.
That seems kind of pointless to me, because if someone has defined a blank option in their `Enum` subclass, they can just use the blank option instead of an empty string.

TL; DR -> Your solution seems optimal. It will be great if you can include a test in your PR.
